packages/website-eslint/src/index.js

```diff
@@ -5,7 +5,8 @@ This saves us having to mock unnecessary things and reduces our bundle size.
 */
 // @ts-check
 
-import { rules } from '@typescript-eslint/eslint-plugin';
+import eslintJs from '@eslint/js';
```

Comment:
i'm not sure if this is the best way to get configs from eslint

Reply:
🤷 it looks like it is!